tests/patmat/i6255b.check

```diff
@@ -0,0 +1 @@
+2: Pattern Match Exhaustivity: _: Expr[Int]
```

Comment:
This is interesting, we don't have the warning before. I'll have a look at it.

Reply:
It made sense because Expr is not sealed anymore. The test case was clearly not exhaustive.

Reply:
I wonder why the error was not there before